ADAL/src/public/ADAuthenticationParameters.h

```diff
@@ -86,4 +87,15 @@ typedef void (^ADParametersCompletion)(ADAuthenticationParameters * _Nullable pa
 + (void)parametersFromResourceUrl:(nonnull NSURL *)resourceUrl
                   completionBlock:(nonnull ADParametersCompletion)completionBlock;
 
+
+/*!
+    Generates default WKWebviewConfiguration with "PkeyAuth/1.0" keyword appended to the UserAgent String.
```

Comment:
I don't think this internal detail is important for the developers, and it does more than just PkeyAuth configuration (we might expand scope even more in the future)
IMO, instead we should tell that this is default recommended configuration that developers should use when they provide custom WKWebView to ADAL + point to the API that let's them provide custom webview.